src/analytics/api/docs/dashboard/exceedances_post.yml

```diff
@@ -6,17 +6,23 @@ parameters:
     in: query
     type: string
     required: true
-    default: airqo
+    default: kcca
   - name: body
     in: body
     type: object
     properties:
       pollutant:
         type: string
-        example: PM 2.5
+        example: pm2_5
       standard:
         type: string
-        example: AQI
+        example: aqi
+      startDate:
+        type: string
+        example: "2021-07-13T21:00:00.000Z"
+      endDate:
+        type: string
+        example: "2021-07-10T21:00:00.000Z"
```

Comment:
Is it intentional that the endDate must be less than the startDate ??
See below results from the swagger tests
![error](https://user-images.githubusercontent.com/37845280/125764080-a1122741-1896-4c92-86a5-014c0272b26a.png)